pkgs/os-specific/linux/new-lg4ff/default.nix

```diff
@@ -13,6 +13,10 @@ stdenv.mkDerivation {
 
   preBuild = ''
     substituteInPlace Makefile --replace-fail "modules_install" "INSTALL_MOD_PATH=$out modules_install"
+    substituteInPlace hid-lg4ff.c --replace-fail "#include \"hid-ids.h\"" "
+    #include \"hid-ids.h\"
+    #define CONFIG_LEDS_CLASS 1
+    "
```

Comment:
Is there a better way of doing this? 
I'd prefer to not patch the code, but I couldn't find anyway to set the flag.
The module doesn't have a kconfig file. we can add it if the `kernel.moduleBuildDependencies` supports getting configs from it.